src/core/keyMap.ts

```diff
@@ -0,0 +1,9 @@
+const keyMap = {
```

Comment:
I don't think this belongs to the core logic, IMO it's more of an UI thing. Does that make sense?